src/quanteda/financial_metrics.py

```diff
@@ -0,0 +1,36 @@
+import pandas as pd
+
+def generate_financial_metrics(random_returns_df, annual_risk_free=0.00):
+    """
+    Calculate financial metrics based on a DataFrame of random returns on time series.
+
+    Parameters
+    ----------
+    random_returns_df : pd.DataFrame
+        DataFrame containing random return values with a datetime index.
+
+    annual_risk_free : float, default is 0.00.
+        Annualized risk-free rate as a decimal (e.g., 0.02 for 2%).
+        
+    Returns
+    -------
+    pd.DataFrame:
+        DataFrame containing financial metrics including count (number of returns), 
+        average return, total return, volatility, and Sharpe ratio.
+
+    Examples
+    --------
+    >>> random_returns_df = generate_return_series(expected_annual_return, 
+                                                   annual_volatility, 
+                                                   freq='H', 
+                                                   length=5, 
+                                                   num_series=3, 
+                                                   dist="normal")
+    >>> annual_risk_free_rate = 0.02
+    >>> result = generate_financial_metrics(random_returns_df, annual_risk_free_rate)
```

Comment:
simplify the example 

```suggestion
    >>> generate_financial_metrics(random_returns_df, annual_risk_free_rate=0.02)
```